### DISCUSSION TITLE
Render problem ?

### DISCUSSION THREAD

**Discussion by Melih4irat** [neuefische/questions#425] [Web]
Render problem in Chrome
in Safari it still works.

Click on the second icon (Projects) in the navigation and then on the project called Projectzero. There you get to the associated tasks. If you now want to scroll into the tasks, there is a render problem or loading problem.

branch: https://github.com/Melih4irat/Capstone/tree/testdnd
folder: pages/projects/[projectid].js

**Comment by actyralf** [comment]
It seems that the `backdrop-filter` property on your `TaskInformation` component causes this behavior. If you remove it, the repaint on scrolling seems to work well.
